jira/resources.py

```diff
@@ -899,7 +900,13 @@ def delete(self, moveFixIssuesTo=None, moveAffectedIssuesTo=None):
         return super(Version, self).delete(params)
 
     def update(self, **args):
-        """Update this project version from the server. It is prior used to archive versions."""
+        """
+        Update this project version from the server. It is prior used to archive versions.
+
+        link to atlassian restapi `documentation`_.
+
+        .. _documentation: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-project-versions/#api-rest-api-2-version-id-get
```

Comment:
```suggestion
        .. _documentation: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-project-versions/#api-rest-api-2-version-id-put
```

The current link points to the `get` method, the suggested link points to the `put` method which is what is used when updating.

Reply:
thx for checking (y)